subsys/bluetooth/host/l2cap.c

```diff
@@ -911,10 +912,11 @@ static int l2cap_chan_le_send_sdu(struct bt_l2cap_le_chan *ch,
 
 static void l2cap_chan_tx_process(struct k_work *work)
 {
+	struct k_work_delayable *dwork = k_work_delayable_from_work(work);
 	struct bt_l2cap_le_chan *ch;
 	struct net_buf *buf;
 
-	ch = CONTAINER_OF(work, struct bt_l2cap_le_chan, tx_work);
+	ch = CONTAINER_OF(dwork, struct bt_l2cap_le_chan, tx_work);
```

Comment:
Could do 

```suggestion
	ch = CONTAINER_OF(k_work_delayable_from_work(work), struct bt_l2cap_le_chan, tx_work);
```

as `dwork` isn't used for any other purpose

Reply:
Done